### ISSUE TITLE
Likert Table radio with reference to text resources has incorrect label (Screen readers)

### ISSUE THREAD

**Issue by lasseklovstad** [Altinn/app-frontend-react#1205]
### Description of the bug

When upgrading newest version of the react frontend i encountered that the radiobutton in the likert table was missing. My playwright tests could not click them. This will impact screen reader users.

![image](https://github.com/Altinn/app-frontend-react/assets/43168787/041af1dc-bc4a-4348-86cd-a731de3611f6)

```json
// Formlayout for likert
{
  "edit": {
    "mode": "likert",
    "filter": [
      { "key": "start", "value": "3" },
      { "key": "stop", "value": "5" }
    ]
  },
  "children": ["formlayout-uniqueid-10"],
  "maxCount": 99,
  "id": "formlayout-uniqueid-9",
  "type": "Group",
  "textResourceBindings": { "leftColumnHeader": "qt-1715-description" },
  "dataModelBindings": { "group": "Questions" }
},
{
  "likertGroup": {
    "edit": {
      "mode": "likert",
      "filter": [
        { "key": "start", "value": "3" },
        { "key": "stop", "value": "5" }
      ]
    },
    "children": ["formlayout-uniqueid-10"],
    "maxCount": 99,
    "id": "formlayout-uniqueid-9",
    "type": "Group",
    "textResourceBindings": { "leftColumnHeader": "qt-1715-description" },
    "dataModelBindings": { "group": "Questions" }
  },
  "optionsId": "options-2097",
  "required": false,
  "id": "formlayout-uniqueid-10",
  "type": "Likert",
  "textResourceBindings": { "title": "likert-row-title" },
  "dataModelBindings": { "simpleBinding": "Questions.Answer" }
}

// Options
[
    {
        "value": "5",
        "label": "op-2941",
        "description": null,
        "helpText": null
    },
    {
        "value": "4",
        "label": "op-2942",
        "description": null,
        "helpText": null
    },
    {
        "value": "3",
        "label": "op-2943",
        "description": null,
        "helpText": null
    },
    {
        "value": "2",
        "label": "op-2944",
        "description": null,
        "helpText": null
    },
    {
        "value": "1",
        "label": "op-2945",
        "description": null,
        "helpText": null
    }
]

// Textresources
{
  "id": "op-2941",
  "value": "I alle eller de fleste fag",
  "variables": null
},
{
  "id": "op-2942",
  "value": "I mange fag",
  "variables": null
},
{
  "id": "op-2943",
  "value": "I noen fag",
  "variables": null
},
{
  "id": "op-2944",
  "value": "I svært få fag",
  "variables": null
},
```

### Steps To Reproduce

1. Run a app with config as described above.
2. And validate the radio button labels.
3. The label seems to: Prioriterer du å bruke tid på skolearbeidet (både arbeid i timene og lekser)? op-2941
4. You can see the option id is in the label which it should not.

### Additional Information

After searching around in the code it seems to be something wronge in the LikertComponent.tsx:

```typescript
<RadioButton
  aria-labelledby={`${rowLabelId} ${colLabelId}`}
  checked={isChecked}
  onChange={handleChange}
  value={option.value}
  label={`${getPlainTextFromNode(text)} ${option.label}`}
  hideLabel={true}
  name={rowLabelId}
  radioId={inputId}
  error={!isValid}
/>;
```
First of all RadioButton in digdir doesn't have aria-labelledby prop. So the label used will be the label prop. Seen in the label prop it doesn't look up the option with the getTextResources function.
